src/pixelator/utils/simplification.py

```diff
@@ -0,0 +1,119 @@
+"""Implementation of the Ramer–Douglas–Peucker line simplification algorithm.
+
+Based on https://github.com/fhirschmann/rdp.
```

Comment:
I think we need to bring in the complete original copyright statement here.

Reply:
Done.